src/test/util/blocktools.h

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) 2019 The Unit-e developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef UNIT_E_TEST_UTIL_BLOCKTOOLS_H
+#define UNIT_E_TEST_UTIL_BLOCKTOOLS_H
+
+#include <test/test_unite_mocks.h>
+
+#include <chain.h>
+#include <random.h>
+#include <uint256.h>
+
+#include <map>
+
+namespace blocktools {
+
+struct BlockIndexFake {
+
+  std::map<uint256, CBlockIndex> block_indexes;
+
+  //! Creates a new CBlockIndex in block_indexes and returns a pointer to it.
+  //!
+  //! The height and pointer to previous block are deduced from the previous
```

Comment:
nit: to `the` previous block

Reply:
Fixed in aa9c9a6542004ded359b1d8bbb1e5505c43df8bb